helios/views.py

```diff
@@ -1379,6 +1388,17 @@ def voters_email(request, election):
 
         tasks.voters_email.delay(election_id = election.id, subject_template = subject_template, body_template = body_template, extra_vars = extra_vars, voter_constraints_include = voter_constraints_include, voter_constraints_exclude = voter_constraints_exclude)
 
+        # get send_to
+        send_to_choices = email_form.fields['send_to'].choices
```

Comment:
I need to get the description of the value. If I get the form value, the text recorded in the log is not very friendly.